catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/reader/GetRecordsMessageBodyReader.java

```diff
@@ -80,6 +81,8 @@ private void buildArguments(CswSourceConfiguration configuration) {
         argumentHolder.put(CswConstants.AXIS_ORDER_PROPERTY, configuration.getCswAxisOrder());
         argumentHolder.put(Metacard.RESOURCE_URI, configuration.getMetacardMapping(Metacard.RESOURCE_URI));
         argumentHolder.put(Metacard.THUMBNAIL, configuration.getMetacardMapping(Metacard.THUMBNAIL));
+        argumentHolder.put(CswConstants.TRANSFORMER_LOOKUP_KEY, TransformerManager.SCHEMA);
```

Comment:
Unit tests were updated for the CswTransformProvider. The unit tests for GetRecordMessageBodyReader doesn't actually need them. Added checks to make sure the new properties are getting set.